compiler/test/compilable/aggr_alignment.d

```diff
@@ -25,7 +25,7 @@ class C2 // overall alignment: max(vtbl.alignof, monitor.alignof, 1, 2)
 enum payloadOffset = C2.bytes.offsetof;
 static assert(C2.int1.offsetof == payloadOffset + 8);
 static assert(__traits(classInstanceAlignment, C2) == size_t.sizeof);
-static assert(__traits(classInstanceSize, C2) == payloadOffset + 12); // no tail padding
+static assert(__traits(classInstanceSize, C2) == payloadOffset + 12 + 4); // with tail padding
```

Comment:
I didn't actually check half of these on 32bit, hope they're ok. :-)

Reply:
Yeah:
```
FAILED targets:
- compilable\aggr_alignment.d
- compilable\test21065.d
- compilable\testfwdref.d
```